### DISCUSSION TITLE
Vercel ärgert mich (solved)

### DISCUSSION THREAD

**Discussion by ostermann-felix** [neuefische/questions#201] [Web] [answered]
Moin. Mein Vercel Deployment baut sich nicht und ich hab das ganze Wochenende geschaut was es sein könnte und hab es nicht gefunden. Hat da jemand einen Tipp wie ich keine Löcher mehr in meinen Schreibtisch beiße?? Folgend mein PR

https://github.com/ostermann-felix/capstone-project/pull/9

**Comment by NikRoe** [comment, marked as answer]
Hey Felix,
can you provide us with a screenshot of the error message you receive when trying to deploy on Vercel or when running `npm run build` in your project?

Thanks in advance

**Reply by NikRoe** [reply]
It looks to me like the import of your `Layout´ component is causing the issue.

In your `_app.js` you import your component from a layout file written with a capital L.
```js
import Layout from '../components/Layout';
```

In your components folder however, the layout file is written with a small l.

Making only case-sensitive changes to files or folders aren't recognised by git. You need to ensure that git is aware of the change you made at some point. 
I found an [article](https://daily-dev-tips.com/posts/git-basics-help-my-case-sensitive-filename-changes-dont-commit/) that might help you with this issue. (There is a part about files and a part about folders. they need to be treated differently)

**Comment by ostermann-felix** [comment]
![Bildschirmfoto 2022-10-04 um 10 08 32](https://user-images.githubusercontent.com/94837451/193767888-b5d8c901-60db-4246-9082-97381d73dac9.png)

**Comment by ostermann-felix** [comment]
![Bildschirmfoto 2022-10-04 um 10 09 22](https://user-images.githubusercontent.com/94837451/193768035-4ac75809-ab43-4099-ae98-69fb5adb7142.png)

**Reply by NikRoe** [reply]
It looks to me like the import of your `Layout´ component is causing the issue.

In your `_app.js` you import your component from a layout file written with a capital L.
```js
import Layout from '../components/Layout';
```

In your components folder however, the layout file is written with a small l.

Let me know if this already fixes the issue and let me know otherwise

**Reply by ostermann-felix** [reply]
Can't find the small l.

![Bildschirmfoto 2022-10-04 um 11 15 20](https://user-images.githubusercontent.com/94837451/193782339-74fcc67b-6395-4178-9bc5-90cefe89a8e7.png)
 This is my Layout component.

**Reply by NikRoe** [reply]
It is the name of the file, not the name of the component that is causing the issue:

![Screenshot 2022-10-04 at 11 18 29](https://user-images.githubusercontent.com/98757391/193783168-36b731ce-de42-44c9-a239-680f7a5abd75.png)

**Reply by ostermann-felix** [reply]
I checked it twice, in my folder its written with a capital L.

**Comment by ostermann-felix** [comment]
![Bildschirmfoto 2022-10-04 um 11 24 14](https://user-images.githubusercontent.com/94837451/193784313-5efa0c66-6c33-4bdc-b34b-2ced59b4ae57.png)
![Bildschirmfoto 2022-10-04 um 11 24 19](https://user-images.githubusercontent.com/94837451/193784318-f697f19d-3c0e-4354-81ab-407d9065ea7c.png)
<img width="365" alt="Bildschirmfoto 2022-10-04 um 11 24 26" src="https://user-images.githubusercontent.com/94837451/193784319-c72d0c57-2f2f-44e7-b5fc-fc16e1dec277.png">

**Reply by NikRoe** [reply]
Making only case-sensitive changes to files or folders aren't recognised by git. You need to ensure that git is aware of the change you made at some point. 
I found an [article](https://daily-dev-tips.com/posts/git-basics-help-my-case-sensitive-filename-changes-dont-commit/) that might help you with this issue. (There is a part about files and a part about folders. they need to be treated differently)

**Comment by ostermann-felix** [comment]
Yes!!! Got it! header.js and layout.js where both not recognized by git. What a shit if the error isn't in the real code but still in the git push... Solved!!

**Reply by NikRoe** [reply]
nice 💪
